aws_lambda_powertools/utilities/streaming/transformations/gzip.py

```diff
@@ -0,0 +1,9 @@
+from gzip import GzipFile
+from typing import IO
+
+from aws_lambda_powertools.utilities.streaming.transformations.base import BaseTransform
+
+
+class GzipTransform(BaseTransform):
```

Comment:
missing docstring to help aid customers understanding its usage and whether additional args can be used.

UX test: Customer imports it, tries to use it, what's the minimum they should know to be productive? they might not have access to the User docs.